### PR TITLE
html and css exercises. 4 html exercises and css 3 image exercise.

### PR DESCRIPTION
**Pull request by dylangoldstock**
Added HTML, css, and JS exercises. Attached is screen shots of the js exercises outputs and the holy grail layout. Edit: Just added more JS exercise outputs and udemy completion.  

jsexercise2
holygrailsc
jsexercise1

jsexercise4

jsexercise3
jsUdemyProgress

### CODE REVIEW

**Comment by dylangoldstock** [comment]
Added js script exercises and attached screen shot. 
jsexercise2

**Review by madhanramu — CHANGES_REQUESTED** [review]
Add description of the PR with screenshots of the output

File: dylangoldstock/javascript/js_exercise2.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                    return b;
+                    
+                }
+
+                if(a1 < b1){
+                    console.log(a, "is closest to 100")
+                    return a;
+                    
+                }
+            }
```
Comment (madhanramu) [inline dylangoldstock/javascript/js_exercise2.html:21]: what will be the output if both a and b are equal? the code should return the output

**Review by madhanramu — CHANGES_REQUESTED** [review]

**Review by dylangoldstock — COMMENTED** [review]
Fixed issues

File: dylangoldstock/javascript/js_exercise2.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                    
+                }
+                if(a1 < b1){
+                    console.log(a, "is closest to 100");
+                    return a;
+                    
+                }
+            }
+            else{
+                console.log("entered values are equal")
```
Comment (madhanramu) [inline dylangoldstock/javascript/js_exercise2.html:22]: Print the number like  console.log(a, "is closest to 100");

File: dylangoldstock/javascript/js_exercise4.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    </h1>
+
+    <ul id="wishlist">
+        
+    </ul>
+
+    <script>
+
+        function add(p){
+            document.getElementById("wishlist").appendChild(p);
```
Comment (madhanramu) [inline dylangoldstock/javascript/js_exercise4.html:24]: the element should be removed from the "products" list if it is added to Wish list

Comment (dylangoldstock) [reply dylangoldstock/javascript/js_exercise4.html:24]: It should work now.

**Review by madhanramu — CHANGES_REQUESTED** [review]

**Review by madhanramu — APPROVED** [review]

**Pull request merged**